oak/server/testdata/key.pub

```diff
@@ -0,0 +1,2 @@
+untrusted comment: minisign public key 199FF2AA1C6A7CDD
```

Comment:
Where is the corresponding private key? How do these need to be re-generated if things change? Please document things so that if anyone else had to make any change and you are not around, they would be able to find their way around.